### PR TITLE
Crafted holy symbol.

### PR DESCRIPTION
**Pull request by Rivet-the-Zombie**
For Spiritual characters who cannot find a holy symbol, this grants the
ability to make one.

### CODE REVIEW

**Comment by Zireael07** [comment]
Yaay!

File: data/json/recipes/recipe_others.json [hunk 1]
```diff
@@ -2110,6 +2110,28 @@
   ]
 },{
   "type" : "recipe",
+  "result": "holy_symbol_wood",
+  "category": "CC_OTHER",
+  "subcategory": "CSC_OTHER_OTHER",
+  "difficulty": 0,
```
Comment (Darkyhard) [inline data/json/recipes/recipe_others.json]: This recipe could probably use fabrication skill. Carving stuff out of wood with a knife is not something that most people do everyday, so you're supposed to learn something from doing it.

**Comment by Chezzo** [comment]
Will this make the quest to go get a spiritual object really easy?

**Comment by RadaRadaRada** [comment]
It's a separate item crafted from wood, so, no.

**Comment by Rivet-the-Zombie** [comment]
Bleh. I screwed up the formatting; will fix asap.

**Pull request merged**